src/org/daisy/dotify/api/formatter/BlockContentBuilder.java

```diff
@@ -89,7 +89,8 @@
 	 * Inserts an expression to evaluate.
 	 * @param exp the expression
 	 * @param t the text properties
+	 * @param numeralStyle the numeral style
 	 */
-	public void insertEvaluate(DynamicContent exp, TextProperties t);
+	public void insertEvaluate(DynamicContent exp, TextProperties t, NumeralStyle numeralStyle);
```

Comment:
I think I would have made this either an `Optional<NumeralStyle>` or allow `null`. Currently you set it to `NumeralStyle.DEFAULT` in formatter.impl if no `number-format` attribute is specified. It works (because `NumeralStyle.DEFAULT.format(x)` happens to be a no-op), but a "default" (a.k.a. "decimal") number style is not strictly the same as "no" number style. Another solution could be to rename `DEFAULT` to `NONE` because the default for `page-number` is not the default for `evaluate`.